module/builder/consensus/builder.go

```diff
@@ -384,14 +384,22 @@ func (b *Builder) getInsertableSeals(parentID flow.Identifier) ([]*flow.Seal, er
 	sealsSuperset := make(map[uint64][]*flow.IncorporatedResultSeal) // map: executedBlock.Height -> candidate Seals
 	sealCollector := func(header *flow.Header) error {
 		blockID := header.ID()
+		// TEMPORARY implementation (unpolished)
+		if blockID == parentID {
+			// Important protocol edge case: There must be at least one block in between the block incorporating a
+			// result and the block sealing the result. This guarantees that a verifier assignment can be computed
+			// without needing information from the block that we are just constructing. Hence, we don't consider
```

Comment:
I'd mention which information we need, e.g. "because the SoR is in the QC for the incorporating block"

Reply:
👍 https://github.com/onflow/flow-go/pull/1307/commits/4967abd0bb339bef1efc55dbe577ae49ebfa0d1a